lib/improver/cli/__init__.py

```diff
@@ -408,11 +412,7 @@ def execute_command(dispatcher, prog_name, *args,
     if dry_run:
         return args
 
-    try:
-        result = dispatcher(prog_name, *args)
-    except Exception as exc:
-        print(exc)
-        raise exc
+    result = dispatcher(prog_name, *args)
```

Comment:
There was originally an intention to have more complex exception handling here, but since there's not any of that at the moment, doing the simple thing is good.